update-conda-pkg.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+if [[ $# -eq 0 ]]; then
+	echo 'usage: ./gh-release version'
+	exit 1
+fi
+
+if [[ ! $(which sha256sum) ]]; then
+	echo "Please, install sha256sum"
+	exit 1
+fi
+
+TAG=$1
+TAG_STR='v${TAG}'
+
+# Update rbc-feedstock
+git clone git@github.com:xnd-project/rbc-feedstock.git
+cd rbc-feedstock
+git remote add upstream git@github.com:conda-forge/rbc-feedstock.git
+git remote add xnd-project git@github.com:xnd-project/rbc-feedstock.git
+git checkout master
+git fetch upstream
+git reset --hard upstream/master
+git push -f -u xnd-project
+
+git branch release-${TAG_STR}
+git checkout release-${TAG_STR}
+
+# + Update ``version`` to ``<tag version>`` and ``sha256`` in ``recipe.yaml``
+sed "s/{% set version = \".*\" %}/{% set version = \"${TAG}\" %}/g" recipe/meta.yaml > recipe/meta.yaml
+
+# XXX: to-do 
+# * For ``sha256``, download the tar-ball and run ``sha256sum`` on it.
+
+git add -u .
+git commit -m "Update to ${TAG_STR}"
+git push -u origin release-${TAG_STR}
+
+msg="
+- Rerender: add a comment containing ``@conda-forge-admin, please rerender``
```

Comment:
We should run conda-smithy locally here as it avoids one more manual step.

Reply:
Or not, as this should be done under `conda-forge/rbc-feedstock`, not under `xnd-project/rbc`